content/posts/aws-powers-read-the-docs.md

```diff
@@ -0,0 +1,132 @@
+and now host documentation for many other languages and frameworks as well.
+
+## Our infrastructure
+
+Read the Docs has a pretty simple architecture,
+because we have a small team of 5 people running a very large web service.
+The major AWS services that we rely on are:
+
+* **EC2**: We run our web servers, web workers, and build servers on EC2 autoscaling groups.
+  This allows us to scale up and down as needed, leading to builders always being available when projects need them.
+* **S3**: We store all of our documentation artifacts in S3.
+  This includes the HTML files, PDFs, and other outputs generated by our build system.
+* **RDS**: Our primary and replica Postgres databases are hosted on RDS.
+  This allows us to scale our database easily, and we rarely have to think about it.
+
+We use a number of other AWS services as well, including Cloudwatch for monitoring, Route 53 for DNS, and many more.
+
+## A few of the projects we host
+
+Here are just a few of the big projects whose docs we build and host:
+
+- [**Godot Engine**](https://docs.godotengine.org/) – the open-source 2D/3D game engine behind thousands of indie titles and student projects.
+- [**Jupyter**](https://docs.jupyter.org/en/latest/) – the open source project that powers Jupyter Notebooks, used by millions of data scientists and educators worldwide.
+- [**Flask**](https://flask.palletsprojects.com/) – the Python web framework powering everything from hobby APIs to production workloads at scale.
+- [**Python Developers Guide**](https://devguide.python.org/) - the official guide for developing Python itself, maintained by the core developers.
+- Tens of thousands of other open-source libraries. Scroll almost any ``requirements.txt`` and you’ll see at least one project whose docs we host.
+
+    <div class="ui padded container">
+        <div class="ui stackable equal width grid">
+            <div class="stretched row">
+                <div class="column">
+                    <div class="ui padded raised large segment">
+                        <i class="fad fa-quotes big icon" aria-hidden="true"></i>
+                        <blockquote class="ui big text">
+                        Read the Docs plays a large part in enabling a small volunteer team to maintain our docs. Without the generous Community plan, we couldn't have afforded anything close to the CI/build resources that we got with Read the Docs.
+                        </blockquote>
+                        <div class="ui small header">
+                            <div class="content">
+                                Max Hilbrunner
+                                <div class="sub header">
+                                    Godot Engine
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+## Features that make Read the Docs unique
```

Comment:
This section feels off topic for a blog post about AWS. I get we probably should have some content describing us if AWS is linking to this post, but I think that content should target users that _have no idea what RTD is_. So, something more like a CTA to learn more, and pointing to a page with a more basic description of RTD would probably be best.

Reply:
That's a great idea, that should fit really well.